test/test_expect.jl

```diff
@@ -0,0 +1,49 @@
+using Test
+using ITensorNetworks
```

Comment:
Please wrap in a module and explicit using statements.

Reply:
I see you wrapped it in a module, but there are still implicit using statements `using Test` and `using ITensorNetworks` rather than explicit using statements `using Test: @test, @testset` and `using ITensorNetworks: ...` (whatever additional types/functions being used by `ITensorNetworks` should be included with the list already started below).

Reply:
Ah yeah! Should be resolved?